firmware/config/boards/core8/prepend.txt

```diff
@@ -0,0 +1,39 @@
+#define ts_show_hip9011 false
+#define ts_show_tle8888 false
+#define ts_show_cj125 false
+#define ts_show_launchcontrol true
+#define ts_show_rollinglaunch true
+#define ts_show_antiLag true
+#define ts_show_rotary false
+#define ts_show_experimental false
+#define ts_show_gdi false
+
+#define ts_show_full_pinout true
+
+#define ts_show_lcd false
+#define ts_show_joystick false
+#define ts_show_egt true
+#define ts_show_gps false
+#define ts_show_etb_pins true
+#define ts_show_analog_divider true
+#define ts_show_spi true
+#define ts_show_sd_card true
+#define ts_show_can_pins true
+#define ts_show_tunerstudio_port true
+#define ts_show_main_relay true
+#define ts_show_main_relay_microRusEFI_message false
+#define ts_show_can2 true
+#define ts_show_software_knock true
+#define ts_show_hardware_simulator true
+
+#define show_test_presets true
+#define show_microRusEFI_presets false
+#define show_Frankenso_presets false
+#define show_Hellen_presets false
+#define show_Proteus_presets false
+
+#define adc_channel_e_enum "NONE", "PA0", "PA1", "PA2", "PA3", "PA4", "PA5", "PA6", "PA7", "PB0", "PB1", "PC0", "PC1", "PC2", "PC3", "PC4", "PC5"
+#define brain_pin_e_enum "NONE","INVALID","PA0","PA1","PA2","PA3","PA4","PA5","PA6","PA7","PA8","PA9","PA10","PA11","PA12","PA13","PA14","PA15","PB0","PB1","PB2","PB3","PB4","PB5","PB6","PB7","PB8","PB9","PB10","PB11","PB12","PB13","PB14","PB15","PC0","PC1","PC2","PC3","PC4","PC5","PC6","PC7","PC8","PC9","PC10","PC11","PC12","PC13","PC14","PC15","PD0","PD1","PD2","PD3","PD4","PD5","PD6","PD7","PD8","PD9","PD10","PD11","PD12","PD13","PD14","PD15","PE0","PE1","PE2","PE3","PE4","PE5","PE6","PE7","PE8","PE9","PE10","PE11","PE12","PE13","PE14","PE15","PF0","PF1","PF2","PF3","PF4","PF5","PF6","PF7","PF8","PF9","PF10","PF11","PF12","PF13","PF14","PF15","PG0","PG1","PG2","PG3","PG4","PG5","PG6","PG7","PG8","PG9","PG10","PG11","PG12","PG13","PG14","PG15"
```

Comment:
those lines look pretty unusual. please help me understand why does Proteus not need them but core8 does?